src/main/webapp/index.html

```diff
@@ -1,46 +1,34 @@
 <!DOCTYPE html>
 <html>
-<head>
-    <meta charset="utf-8"/>
+  <head>
+    <meta charset="utf-8" />
     <meta
-            content="width=device-width, initial-scale=1, shrink-to-fit=no"
-            name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+      name="viewport"
     />
-    <meta content="STEP Capstone: Techish" name="description"/>
-    <meta content="Samuel Sandoval and Hadley Shapland" name="author"/>
+    <meta content="STEP Capstone: Techish" name="description" />
+    <meta content="Samuel Sandoval and Hadley Shapland" name="author" />
     <title>Techish</title>
-    <link href="css/styles.css" rel="stylesheet"/>
-</head>
-<body onload="loadMatches();">
-<!-- navigation bar -->
-<nav class="navbar navbar-expand-md">
-    <a class="navbar-brand" href="#">techish</a>
-    <button
-            class="navbar-toggler navbar-dark"
-            data-target="#main-navigation"
-            data-toggle="collapse"
-            type="button"
-    >
-        <span class="navbar-toggler-icon"></span>
-    </button>
-    <div class="collapse navbar-collapse" id="main-navigation">
-        <ul class="navbar-nav">
-            <li class="nav-item" id="logging">
-                <a class="login-text" href="#" id="login-link"></a>
-            </li>
-        </ul>
-    </div>
-</nav>
-<h1>Techish Logged-In Homepage</h1>
+    <link href="css/styles.css" rel="stylesheet" />
+  </head>
+  <body>
+    <!-- navigation bar -->
+    <div class="nav-item" id="logging">
```

Comment:
It's not as clear to me what logging represents. This could be confused with logging application data (e.g. console.log). maybe consider a different name?